chart_test.sh

```diff
@@ -55,6 +59,20 @@ main() {
             --no-lint)
                 no_lint=true
                 ;;
+            --chart-all)
+                CHART_ALL=true
+                export CHECK_VERSION_INCREMENT=false
```

Comment:
Please use local variables here to be consistent. This block should only deal with CLI parsing. Export `CHECK_VERSION_INCREMENT` further below. As far a I can see, only `CHECK_VERSION_INCREMENT` needs to be an env var.

Reply:
sure